sdk/sdk-api/src/main/java/mil/emp3/api/MilStdSymbol.java

```diff
@@ -1630,4 +1632,21 @@ public IEmpBoundingBox getFeatureBoundingBox() {
 
         return bBox;
     }
+
+    public void setIconColor(IGeoColor color) {
```

Comment:
Currently this method is in MilStdSymbol. Also could place it in MilStd2525SinglePoint. How is RII planning to use this functionality.

Reply:
This is the correct place.  MilStd2525SinglePoint is a map engine module method.  We want this for all map engines, not just WW.